dsc_lib/src/discovery/command_discovery.rs

```diff
@@ -50,7 +51,10 @@ impl CommandDiscovery {
                     let path = entry.path();
                     if path.is_file() {
                         let file_name = path.file_name().unwrap().to_str().unwrap();
-                        if file_name.to_lowercase().ends_with(".dsc.resource.json") {
+                        let file_name_lowercase = file_name.to_lowercase();
+                        if file_name_lowercase.ends_with(".dsc.resource.json") |
```

Comment:
Should you use the double bar `||` logical OR operator instead of the bitwise OR operator here?